StephaneGrabsia/uQuizz
# Student main page front v4.0.0

This PR connects the back and the front for the student main page

It allows to fetch the list of exercises and actualize it regularly. It also allows to modify the statement printed depending on the exercise selected.  
It also allows the submission of a solution, and display the output and the traceback.
To test it manually:
- Docker compose up 
- Create a professor, a classroom, and exercises
- Go to localhost:8080/student and check if the list of exercises is correctly shown
- Try to change the selected tab in the app bar
- Code a solution, and submit it
- Look if the output and the traceback have been correctly displayed
